README.md

```diff
@@ -1,15 +1,26 @@
-# Inferno Test Writing Tutorial
+# Inferno Template Test Kit
 
-This repository provides a step-by-step tutorial for users interested in writing
-FHIR tests using the [Inferno Framework](https://inferno-framework.github.io/inferno-core/).
+Inferno template [Inferno](https://github.com/inferno-community/inferno-core) Test Kit
+for FHIR testing.
 
-To get started, visit the [Tutorial Wiki](https://github.com/inferno-training/inferno-tutorial/wiki).
+## Instructions for Developing Your Test Kit
 
-This test kit was generated from the
-[Inferno Test Kit Template](https://github.com/inferno-framework/inferno-template).
+Refer to the Inferno documentation for information about [setting up
+your development environment and running your Test Kit](https://inferno-framework.github.io/docs/getting-started/).
+
+More information about what is included in this repository can be [found here](https://inferno-framework.github.io/docs/getting-started/repo-layout-and-organization.html).
+
+## Documentation
+- [Inferno documentation](https://inferno-framework.github.io/docs/)
+- [Ruby API documentation](https://inferno-framework.github.io/inferno-core/docs/)
+- [JSON API documentation](https://inferno-framework.github.io/inferno-core/api-docs/)
+
+## Example Inferno Test Kits
+
+A list of all Test Kits registered with the Inferno Team can be found on the [Test Kit Registry](https://inferno-framework.github.io/community/test-kits.html) page.
 
 ## License
-Copyright 2022 The MITRE Corporation
+Copyright 2024 TODO
```

Comment:
TODO comment here might not supposed to be here? Unless it is a placeholder to just show that copyright company would go here in a developed test kit?